code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -1709,23 +1709,23 @@ var/list/blind_victims = list()
 //RUNE XI
 /datum/rune_spell/blood_cult/seer
 	name = "Seer"
-	desc = "See the invisible, the dead, the concealed. If you give them a writing sheet, they may relay a message to you."
-	desc_talisman = "For a few seconds, you may see the invisible, the dead, the concealed. If you give them a writing sheet, they may relay a message to you."
+	desc = "See the invisible, the dead, the concealed, as well as the living's propension to serve our agenda."
+	desc_talisman = "For a whole minute, you may see the invisible, the dead, the concealed, as well as the living's propension to serve our agenda."
 	Act_restriction = CULT_ACT_I
 	invocation = "Rash'tla sektath mal'zua. Zasan therium viortia."
 	rune_flags = RUNE_STAND
 	talisman_uses = 5
 	word1 = /datum/runeword/blood_cult/see
 	word2 = /datum/runeword/blood_cult/hell
 	word3 = /datum/runeword/blood_cult/join
-	page = "This rune grants you the ability to see the invisible, including observers and concealed runes and structures. The talisman version has 5 uses, which grant you the ability for 8 seconds each. Remember that runes can still be activated while they are concealed! "
+	page = "This rune grants you the ability to see the invisible, including observers and concealed runes and structures. The talisman version has 5 uses, which grant you the ability for an entire minute each, allowing you as well to see how likely each crew members are to accept conversion. Remember to activate the talisman where no one can hear you first. Remember also that runes can still be activated while they are concealed! "
```

Comment:
entire seems extraneous
"each crew member **is**" to accept conversion, use "is" for singular

Reply:
Honestly this whole block of text is awkwardly phrased, with two remembers and separating the part about willingness, just rewrite it

`This rune grants the ability to see invisible ghosts, runes, and structures. It also reveals the willingness of crew members to accept conversion. You can activate runes while they are concealed. In talisman form, it has five uses and lasts for a minute each. Activate the talisman before moving into a public.`